src/ot_orchestration/dags/config/genetics_etl.yaml

```diff
@@ -1,21 +1,78 @@
-- id: "ot_gene_index"
-- id: "ot_variant_index"
-- id: "ot_variant_to_gene"
-  prerequisites:
-    - "ot_variant_index"
-    - "ot_gene_index"
-- id: "ot_colocalisation_ecaviar"
-- id: "ot_colocalisation_coloc"
-- id: "ot_locus_to_gene_train"
-  prerequisites:
-    - "ot_variant_index"
-    - "ot_variant_to_gene"
-    - "ot_colocalisation_ecaviar"
-    - "ot_colocalisation_coloc"
-- id: "ot_locus_to_gene_predict"
-  prerequisites:
-    - "ot_locus_to_gene_train"
-    - "ot_variant_index"
-    - "ot_variant_to_gene"
-    - "ot_colocalisation_ecaviar"
-    - "ot_colocalisation_coloc"
+cluster_name: otg-etl
+gwas_catalog_manifests_path: gs://gwas_catalog_data/manifests
+l2g_gold_standard_path: gs://genetics_etl_python_playground/input/l2g/gold_standard/curation.json
+release_dir: gs://genetics_etl_python_playground/releases/24.09
+
+steps:
+  - id: study_validation
+    params:
+      study_index_path:
+        - gs://gwas_catalog_data/study_index
```

Comment:
The same way we are versioning Finngen (R11), should we version the other datasources?

Reply:
As a matter effect, we do not use the versioning there for now, although if we want to update them it would be great to do that